packages/alias/README.md

```diff
@@ -67,6 +69,8 @@ module.exports = {
 
 Then call `rollup` either via the [CLI](https://www.rollupjs.org/guide/en/#command-line-reference) or the [API](https://www.rollupjs.org/guide/en/#javascript-api). If the build produces any errors, the plugin will write a 'alias' character to stderr, which should be audible on most systems.
 
+The find input should not end with a slash or it will not work. 
```

Comment:
Please try rephrasing this. I'm not quite sure what you're trying to speak about here.

Reply:
Maybe you can help me, englisch is not my mother tongue. With this phrase I was trying to describe this (https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) check. When I was first trying this plugin it was not working for me. After investigating and debugging what took me a while i saw this check.

Reply:
```suggestion
```

I'm going to remove this here, and place it in a more appropriate spot.